components/react/src/styles/main.scss

```diff
@@ -79,25 +49,10 @@ div.detailsGrid {
   border-bottom: 1px solid lightgray;
 }
 
-.row {
-  max-width: 1140px;
-  margin-left: 4px;
-  margin-right: 4px;
-}
-
-/* SELECTOR COMPONENT STYLING */
-
-.card-body {
-  max-width: 350px;
-  margin: 8px;
-}
-
-/* PROJECT COMPONENT STYLING */
+/* COMPONENT STYLING */
 
-@media screen and (min-width: 500px) {
-  div.projectCol1 {
-    border-right: 1px solid lightgray;
-  }
```

Comment:
I'll have to see how this looks with all the other changes, but removing this media query will potentially look weird on a browser at mobile width.